react/src/components/MapControlBar/MapControlbar.tsx

```diff
@@ -143,6 +162,15 @@ const MapControlbar: React.FC<Props> = ({ activeProject, isPublicView }) => {
           </div>
         )}
       </div>
+      <Flex justify="center" gap="small">
+        <Button className={styles.taggitButton}
+          data-testid="taggit-button"
+          type="primary"
+          onClick={() => navigateToCorrespondingTaggitGallery()}
+        >
+          View Project in Taggit
```

Comment:
```suggestion
          View in Taggit
```

I believe we're avoiding Project as has specific meaning in DesignSafe.